lib/mix/tasks/coh.context.ex

```diff
@@ -0,0 +1,187 @@
+defmodule Mix.Tasks.Coh.Context do
+  @shortdoc "Generates a context with functions around an Ecto schema representing a User"
+
+  @moduledoc """
+  Generates a context with functions around an Ecto schema representing a User.
+      mix coh.context Accounts name:string email:string
+  """
+  
+  # Code shamelessly stolen from the Phx default context generator with minor changes
+
+  use Mix.Task
+
+  alias Mix.Phoenix.{Context}
+  alias Mix.Tasks.Phx.Gen
+
+  @switches [binary_id: :boolean, table: :string, web: :string,
+             schema: :boolean, context: :boolean, context_app: :string]
+
+  @default_opts [schema: true, context: true]
+
+  # Hardcoded schema name:
+  @schema_name "User"
```

Comment:
The default schema name (if the --schema option is not provided) should be the last name of `Module.split Config.user_schema`